data/json/terrain.json

```diff
@@ -300,6 +314,7 @@
         "type" : "terrain",
         "id" : "t_dirt",
         "name": "dirt",
+        "description": "It's dirt. Looks like some fine soil for tillage, or dug out for construction projects.",
```

Comment:
Recommend switching `Looks like some fine soil for tillage or dug out...` to something like `Could be tilled and planted with crops, or dug out...`

Reply:
More for flavor than dry explanation